db/scripts/find_cured.py

```diff
@@ -88,13 +89,15 @@ def find_cured(today_datetime, state):
     cursor.execute(queries.get_cured(election))
     output = cursor.fetchall()
 
-    # for each cured entry, add to cured_db
+    # for each cured entry, add to cured_db and remove from rejected_db
     for entry in output:
       cursor.execute(schema.add_to_cured(cured_db, entry))
+      cursor.execute(schema.remove_cured_from_rejected(rejected_db, entry))
+      mydb.commit()
 
     # query the current day for any new rejected that are not cured
     print("Getting today's rejected ballots from main table")
-    cursor.execute(queries.get_today_rejected(election, today_datetime, cured_db))
+    cursor.execute(queries.get_rejected(election, cured_db))
     output = cursor.fetchall()
```

Comment:
Are get_rejected and get_today_rejected the same thing just with different names?

Reply:
Yes, they are the same thing. I realized that `get_today_rejected` didn't actually use `today_datetime`, so the query was just searching through all rejected ballots. Thus, I changed the name to `get_rejected` to be consistent with the logic.